pkg/syncer/apis/edge/register.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2023 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package edge
+
+const (
+	GroupName = "edge.kcp.io"
```

Comment:
1. We keep a 1:1 correspondence between API group and source directory.
2. The API group source directories appear under https://github.com/kcp-dev/edge-mc/tree/main/pkg/apis .

Reply:
Thanks. I will move it to `/pkg/apis`. 

> for example, because you want to make a distinction between resources that are part of the interface of edge-mc vs. resources that are internal

Yes, this is also what I was wondering if syncer related APIs are part of edge-mc interface or not. EdgeSyncConfig can be referred by edge-mc controllers (PlacementTranslater?) if my understanding is correct. It makes sense for the API to be in the same api groups as edge-mc.